### PR TITLE
Allow Fleet Status to report correct nucleus  version

### PR DESCRIPTION
**Pull request by rawalexe**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/examples/sample.ggLitePython/README.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ This demo has two key requirements for it to work.
...
 
 - As a python demo it requires the python3 interpreter (if not already
-  installed) as a well as a python3 virtual environment support for a isolated
-  dependency management.
+  installed) as a well as a python3 virtual environment for isolated dependency
+  management.
 
-- As an alternative method you may deploy the following recipe as a seprate
-  deployment before deploying given python demo.
+- Alternatively, you may deploy the following recipe deployment before deploying
```
Comment (sameerzuberi) [inline docs/examples/sample.ggLitePython/README.md:15]:
```suggestion
- Alternatively, you may deploy the following recipe before deploying
```

File: docs/examples/sample.ggLitePython/README.md [hunk 2]
```diff
@@ -9,11 +9,11 @@ This demo has two key requirements for it to work.
...
+  given python demo.
```
Comment (sameerzuberi) [inline docs/examples/sample.ggLitePython/README.md:16]:
```suggestion
  the given python demo.
```

File: docs/examples/sample.ggLitePython/README.md [hunk 3]
```diff
@@ -59,8 +59,14 @@ You may deploy the componet the following ways:
...
 
-Once the deployment is successful running the following command to check the log
-and see the list of buckets in your S3 that exist in your aws account.
+The logs for following the deployment status can be found by:
+
+```shell
+$ journalctl -xeau ggl.core.ggdeploymentd.service
+```
+
+Once successful, use following command to check the log and see the list of
```
Comment (sameerzuberi) [inline docs/examples/sample.ggLitePython/README.md:68]:
```suggestion
Once successful, use the following command to check the logs and see the list of
```

File: docs/examples/sample.ggLitePython/README.md [hunk 4]
```diff
@@ -59,8 +59,14 @@ You may deploy the componet the following ways:
...
+buckets names in your AWS S3 that exist in your aws account.
```
Comment (sameerzuberi) [inline docs/examples/sample.ggLitePython/README.md:69]:
```suggestion
bucket names that exist in your AWS account within S3.
```

File: gg-fleet-statusd/bin/gg-fleet-statusd.c [hunk 5]
```diff
@@ -4,6 +4,10 @@
 
 #include "gg_fleet_statusd.h"
 #include <ggl/error.h>
+#include <ggl/version.h>
+
+__attribute__((visibility("default"))) const char *argp_program_version
```
Comment (AniruddhaKanhere) [inline gg-fleet-statusd/bin/gg-fleet-statusd.c:9]: Why do you need this?

**Review by AniruddhaKanhere — CHANGES_REQUESTED** [review]

File: gg-fleet-statusd/bin/gg-fleet-statusd.c (hunk 5 above)
Comment (rawalexe) [reply gg-fleet-statusd/bin/gg-fleet-statusd.c:9]: If you do `./bin/gg-fleet-statusd --version` you will get the same value as nucleus

**Review by sameerzuberi — APPROVED** [review]

**Review by AniruddhaKanhere — APPROVED** [review]

**Pull request merged**